static/javascript/chameleon_game.js

```diff
@@ -1,15 +1,123 @@
 var score = 0;
 
-function change_color(div_id) {
+function change_color(div_id, div_id2) {
+	// Function changes the Octocat color/position and game background and updates score whenever the Octocat is clicked
     document.getElementById("confirm-submit").innerHTML = "Never mind...";
-    document.getElementById(div_id).style.backgroundColor = random_color();
+	rando = Math.random() * 100;
+    document.getElementById(div_id).style.backgroundColor = get_BGcolor(rando);
+	document.getElementsByClassName(div_id2)[1].style.color = get_OCcolor(rando);
+	document.getElementsByClassName(div_id2)[1].style.position = "relative";
+	document.getElementsByClassName(div_id2)[1].style.top = random_percent_top();
+	document.getElementsByClassName(div_id2)[1].style.left = random_percent_left();
     score++;
+	document.getElementById('scores').innerHTML = 'Your Current Score: ' + score.toString();
 }
 
-function random_color() {
-  var c = '';
-  while (c.length < 6) {
-    c += (Math.random()).toString(16).substr(-6).substr(-1);
-  }
-  return '#'+c;
+function get_BGcolor(rando){
```

Comment:
get_BGcolor and get_OCcolor look hella sketch. It'd be better to have an array color_array of hexcolor codes and then use rando%color_array as the index into that array to select the color.